test/gamma.jl

```diff
@@ -118,6 +120,38 @@
         @test loggamma(Inf + Inf*im) === loggamma(NaN + 0im) === loggamma(NaN*im) === NaN + NaN*im
     end
 
+    @testset "BigFloat" begin
+        # test cases (values taken from WolframAlpha)
+        @test loggamma(big(3.099)) ≈ 0.786413746900558058720665860178923603134125854451168869796
```

Comment:
Need to add an `rtol=1e-70` or something … otherwise the default is to check that only half the digits match.

Reply:
I improved the tests and ensured that the ground truth is also parsed as a BigFloat.